applications/ShapeOptimizationApplication/python_scripts/value_logger_base.py

```diff
@@ -51,11 +66,11 @@ def FinalizeLogging( self ):
 
     # --------------------------------------------------------------------------
     def GetValue( self, key, iteration ):
```

Comment:
The name `GetValue` and `GetValueHistory` are now a little bit misleading because you can get other things then the value as well, e.g. the relative change. Not a big issue though

Reply:
True. I tried to make it a bit more readable by changing the keys in the history